charmcraft.yaml

```diff
@@ -1,45 +1,96 @@
-# Copyright 2024 Canonical Limited
+# Copyright 2022 Canonical Ltd.
```

Comment:
nit: typo in the date?

Reply:
NVM, I'm gonna merge this to un block myself, and update this in subsequent PR if need be.